R/sce_to_anndata.R

```diff
@@ -1,8 +1,11 @@
 #' Convert SingleCellExperiment objects to AnnData file stored as HDF5 file
 #'
 #' @param sce SingleCellExperiment object to be converted to AnnData as an HDF5 file
-#' @param anndata_file Path to output AnnData file. Must be an `.h5` or `.hdf5`
+#' @param anndata_file Path to output AnnData file. Must end in `.h5`, `.hdf5`, or `.h5ad`
 #' @param x_assay_name Name of assay in SCE object to save as X in AnnData. Default is "counts".
+#' @param compression Type of compression to use when exporting hdf5 files. Options are
+#'   "none", "gzip", or "lzf". Default is "gzip".
+#' @param ... Any arguments to be passed into zellkonverter::writeH5AD or zellkonverter::SCE2AnnData
```

Comment:
I know this gets passed internally to `SCE2AnnData`, but we don't use it, so it seems confusing to name it here.
```suggestion
#' @param ... Any arguments to be passed into zellkonverter::writeH5AD
```